server/src/main/java/org/apache/iotdb/db/service/TSServiceImpl.java

```diff
@@ -886,11 +837,8 @@ public TSFetchResultsResp fetchResults(TSFetchResultsReq req) {
           QueryResourceManager.getInstance().endQuery(req.queryId);
           queryId2DataSet.remove(req.queryId);
         }
-        TSFetchResultsResp resp =
-            getTSFetchResultsResp(
-                getStatus(
-                    TSStatusCode.SUCCESS_STATUS,
-                    "FetchResult successfully. Has more result: " + hasResultSet));
+        TSFetchResultsResp resp = RpcUtils.getTSFetchResultsResp(
+            TSStatusCode.SUCCESS_STATUS, "FetchResult successfully. Has more result: " + hasResultSet);
```

Comment:
do we really need such a long string as the message?

Reply:
This message is not needed, removed